src/views/Viewer.vue

```diff
@@ -348,6 +348,10 @@ export default {
 			return ['image/jpeg', 'image/png', 'image/webp'].includes(this.currentFile?.mime)
 		},
 
+		showsMimeImage() {
```

Comment:
* let's remove this (CSS hide logic) in favor of setting NcModal.name (see https://github.com/IONOS-Productivity/nc-viewer/tree/review/fb/feat/add_default_previewer/default_component)